### ISSUE TITLE
[Bug]: [GroupedList]: ARIA attribute cannot be used, add a role attribute or use a different element: aria-label

### ISSUE THREAD

**Issue by marcosmoura** [microsoft/fluentui#23784]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
- Package version(s): v7 and v8
- Browser and OS versions: Chrome and Edge, NVDA, JAWS, and Narrator
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://aka.ms/fluentpen

### Bug Description

GroupedList is flagged by Accessibility Insights for a MAS 4.1.2 violation caused by improper aria attributes.

While this was detected in our application, it is also detected on the Usage sample page of the Fluent GroupedList documentation:
Fluent UI - Controls - React - GroupedList

## Actual behavior:
Accessibility Insights finds bugs on GroupedList component.

## Expected behavior:
Accessibility Insights should find no bugs on GroupedList component.

### Requested priority

Low

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by espeo196** [comment]
Hi, did you end up checking in a fix to this bug? I see there's another issue already logged for this from May.
